Directory.build.props

```diff
@@ -1,6 +1,7 @@
 <Project>
 	<PropertyGroup>
 		<Deterministic>true</Deterministic>
-		<LangVersion>9.0</LangVersion>
+		<LangVersion>10.0</LangVersion>
+		<ImplicitUsings>true</ImplicitUsings>
```

Comment:
This is the default for new projects in their template. It maybe feels weird but I think its pretty nice when you get used to it. The list depends on your project's SDK, for normal .NET things it is:

```
System
System.Collections.Generic
System.IO
System.Linq
System.Net.Http
System.Threading
System.Threading.Tasks
```

Reply:
`<nullable>` is also enabled by default. I haven't done that yet that in this PR because there will be some clean-up.

Reply:
The `ImplicitUsings` scares me, but not enough to get use to the new world

Reply:
It's pretty questionable. Like they better be really careful about adding new things in those namespaces... but 99% of the time you end up having all of those anyway so it's really not much of a change.

Reply:
Mostly I just don't want to fight it, but it does make writing a new file a little less annoying (which is a niche concern)